kubernetes/machine-class.yaml

```diff
@@ -15,15 +15,59 @@
 apiVersion: machine.sapcloud.io/v1alpha1
 kind: MachineClass
 metadata:
-  name: test-mc
-  namespace: default # Namespace where the controller would watch
-providerSpec:
-  storageClassName: test-storage-class
-  pvcSize: "10Gi"
-  sourceURL: source-image-url
-  cpus: "1"
-  memory: "4096M"
+  name: test-machine-class
   namespace: default
-secretRef: # If required
+providerSpec:
+  region: local
+  zone: local-1
+  resources:
+    requests:
+      cpu: 1
+      memory: 4096Mi
+    limits:
+      cpu: 2
+      memory: 8Gi
+  rootVolume:
+    pvc:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 8Gi
+      storageClassName: standard
+    source:
+      http:
+        url: https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
```

Comment:
let's change to bionic since os extension supports 18.04 >=
`https://cloud-images.ubuntu.com/bionic/current/bionic-server-cloudimg-amd64.img`

Reply:
Fixed